examples_rclcpp/include/action_client/switcher.hpp

```diff
@@ -0,0 +1,48 @@
+#ifndef EXAMPLES_RCLCPP_ACTION_CLIENT_SWITCHER_HPP_
+#define EXAMPLES_RCLCPP_ACTION_CLIENT_SWITCHER_HPP_
+
+#include <memory>
+#include <string>
+#include <utility>
+
+#include <rclcpp/rclcpp.hpp>
+#include <rclcpp_action/rclcpp_action.hpp>
+
+#include "examples_msgs/action/led.hpp"
+
+
+namespace robotis
+{
+class Switcher : public rclcpp::Node
+{
+ public:
+  explicit Switcher(const int32_t input_value);
+  virtual ~Switcher(){};
+
+  void send_goal();
+
+  bool is_goal_done() const;
+
+  int32_t user_custom_value_ = 5;
+
+ private:
+  void print_message_info();
+
+  void goal_response_callback(
+    std::shared_future<rclcpp_action::ClientGoalHandle<examples_msgs::action::Led>::SharedPtr> future);
```

Comment:
Too long lines. Please check word count

Reply:
Modified long lines to  short lines and push